source/_integrations/hunterdouglas_powerview.markdown

```diff
@@ -31,6 +33,88 @@ There is currently support for the following device types within Home Assistant:
 
 {% include integrations/config_flow.md %}
 
+## Supported Shades
+
+Known devices and their supported functionality is listed below.
+
+> :warning: Devices listed as **Under Development** may perform basic or slightly odd functionality until **Full** support is implemented.\
+> We ask you wait patiently for Full support before raising support tickets for other models.\
+> \
+> If however you're shade is not listed below please raise a support request for them to be added.
+
+| Type | Common Name                        | Capabilities                    | Support           |
```

Comment:
Generally we don't want to get support requests in the issue queue.  We only want people to open an issue if there is an actual bug of flaw.  It would be best to avoid any language that implies its a place where they can go to get support as we want to send people to the community forums for support.

Reply:
If its a feature request we send users to https://community.home-assistant.io/c/feature-requests. If its support https://discord.gg/c5DvZ4e or the forums

But in this case they should find the urls elsewhere since they may change in the future and we don't want to maintain them here as well. 

Ideally we just avoid saying that support is available or something is supported as it implies that the issue queue is a support queue which we want to avoid.  Usually I try to say things are `tested` or `confirmed working`

Reply:
how do i tell what release the tdbu suport will be in - ie the table will need to be updated for the correct next branch

Reply:
rewording commited now